content/en/real_user_monitoring/mobile_and_tv_monitoring/integrated_libraries/ios.md

```diff
@@ -0,0 +1,52 @@
+---
+title: iOS and tvOS Libraries for RUM
+code_lang: ios
+type: multi-code-lang
+code_lang_weight: 10
+aliases:
+- /real_user_monitoring/ios/integrated_libraries/
+further_reading:
+- link: https://github.com/DataDog/dd-sdk-ios
+  tag: "Source Code"
+  text: Source code for dd-sdk-ios
+- link: /real_user_monitoring/mobile_and_tv_monitoring/integrated_libraries
+  tag: Documentation
+  text: Integrated Libraries
+---
+
+This page lists integrated libraries you can use for iOS and tvOS applications.
+
+## Alamofire
+
+Starting from version `2.5.0`, the RUM iOS SDK can automatically track [Alamofire][1] requests.
+
+1. Configure RUM monitoring by following the [Setup][2] guide.
+2. Enable `URLSessionInstrumentation` for `Alamofire.SessionDelegate`:
+
+```swift
+import Alamofire
+import DatadogRUM
+
+URLSessionInstrumentation.enable(with: .init(delegateClass: SessionDelegate.self))
```

Comment:
/improvement

have seen some customer asking where is `SessionDelegate` type.

We should make sure from docs that it clearly states that `SessionDelegate` is a customer type and implemented in their app.

Reply:
It is Alamofire's type 🤔 💭. Which is already covered in:
> 2. Enable `URLSessionInstrumentation` for `Alamofire.SessionDelegate`:

I can make it cleaner by:
```suggestion
URLSessionInstrumentation.enable(with: .init(delegateClass: Alamofire.SessionDelegate.self))
```
WDYT @ganeshnj ?

Reply:
Ultimately, I decided to add `Alamofire.` namespace, same as for Apollo. I also considered linking to [`Alamofire.SessionDelegate` doc](https://github.com/Alamofire/Alamofire/blob/master/Documentation/AdvancedUsage.md#sessiondelegate), but decided to not do it as external links are likely to change making our docs out-of-sync.